### DISCUSSION TITLE
How to migrate nuclei templates from the old syntax to the new syntax

### DISCUSSION THREAD

**Discussion by Caratpine** [projectdiscovery/nuclei#5185] [Q&A]
Hi masters.
We have made a template management ourselves, with about 2,000 templates. The nuclei version we use is 2.8.8. Recently we want to upgrade nuclei to the latest version, but because the templates we maintain are not new syntax, we need to migrate it. , are there any good solutions or tools? Thank you.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
